core/src/main/resources/defaults/blueprints/cdp-data-engineering.bp

```diff
@@ -156,22 +198,6 @@
           }
         ]
       },
-      {
-        "refName": "hue",
-        "serviceType": "HUE",
```

Comment:
Are you sure Hue needs to be removed?

Reply:
Yes, because it doesn't work with Hive on Tez at the moment. There is a possibility that we may ship DAS instead of Hue in DE template.

Reply:
OK, thanks.